controllers/refresolver/ingress/ingress.go

```diff
@@ -52,7 +52,7 @@ func NewReferenceResolver(gslb *k8gbv1beta1.Gslb, k8sClient client.Client) (*Ref
 	}
 
 	if len(ingressList) != 1 {
-		return nil, fmt.Errorf("exactly one Ingress resource expected but %d were found", len(ingressList))
```

Comment:
This is unrelated, but since I was touching error messages I make it more consistent by using an integer both for expected and found ingresses